index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
move your index.html and resume to your pages directory. You should also move your emoji into your assets directory and your styles.css into a styles directory. This will keep everything organized.